src/pages/exam.tsx

```diff
@@ -102,7 +106,21 @@ const Exam = () => {
       notifyError('¡Selecciona la respuesta!')
       return
     }
-    window.location.href = '/exam' // Pending
+
+    let points = 0
+    let average = 0
+    questions.forEach((question: Question) => {
+      const selectedOption = selectedOptions[question.question as keyof typeof selectedOptions]
+      const correctOption = question.options.find((option) => option.points > 0)
+      if (selectedOption === correctOption?.option) {
+        points += correctOption.points
+        average = points / questions.length
+      }
+    })
```

Comment:
What about we make this sum when the user press "Next" button?

Reply:
Yeah, i was thinking about it too, but, what about the modal?? Remember that? we need to show the modal! or what about that ?

Reply:
And the final calculation logic can be on Result component

Reply:
Oh ok, so what you recommend for this, a new function to storage the same logic about the result ? cuz, you know Next and surveEnd are differents functions.

Reply:
In general, you should use the context too

Reply:
I need to test this.

Reply:
Done